examples/complete/main.tf

```diff
@@ -0,0 +1,152 @@
+}
+
+##############################################################################
+# Service Credentials
+##############################################################################
+
+resource "ibm_resource_key" "service_credentials" {
+  count                = length(var.service_credentials)
+  name                 = var.service_credentials[count.index]
+  resource_instance_id = module.event_streams.id
+  tags                 = var.resource_tags
+}
+
+# The current implementation of the secrets manager module will not allow us to generate secrets from values that are unknown in advance
+# The Secrets manager provider support is coming soon and the module can be refactored to handle this.
+# Until then you need to apply without the following blocks then uncomment them and apply in a followup apply.
+# This will not do in a production environment and is just here for inspiration
+
+#locals {
+#  # tflint-ignore: terraform_unused_declarations
+#  users = [for index, user in var.service_credentials :
+#    {
+#      name : user
+#      username : ibm_resource_key.service_credentials[index].credentials["connection.https.authentication.username"],
+#      password : sensitive(ibm_resource_key.service_credentials[index].credentials["connection.https.authentication.password"]),
+#      cert : sensitive(base64decode(ibm_resource_key.service_credentials[index].credentials["connection.https.certificate.certificate_base64"]))
+#    }
+#  ]
+#}
+
+#module "secrets_manager_service_credentials_user_pass" {
+#  providers = {
+#    restapi = restapi.sm
+#  }
+#  source                  = "git::https://github.ibm.com/GoldenEye/secrets-manager-secret-module?ref=2.3.6"
+#  count                   = length(local.users)
+#  region                  = var.region
+#  secrets_manager_guid    = ibm_resource_instance.secrets_manager.secrets_manager_guid
+#  secret_group_id         = module.secrets_manager_secrets_group.secret_group_id
+#  secret_name             = local.users[count.index].name
+#  secret_description      = "Event Streams Service Credential"
+#  secret_username         = local.users[count.index].username
+#  secret_payload_password = local.users[count.index].password
+#}
+#
+#module "secrets_manager_service_credentials_cert" {
+#  providers = {
+#    restapi = restapi.sm
+#  }
+#  depends_on = ["ibm_resource_key.service_credentials"]
+#  source                  = "git::https://github.ibm.com/GoldenEye/secrets-manager-secret-module?ref=2.3.6"
+#  count                   = length(var.service_credentials)
+#  region                  = var.region
+#  secrets_manager_guid    = ibm_resource_instance.secrets_manager.secrets_manager_guid
+#  secret_group_id         = module.secrets_manager_secrets_group.secret_group_id
+#  secret_name             = local.users[count.index].name
+#  secret_description      = "Event Streams Service Credential Certificate"
+#  imported_cert              = true
+#  imported_cert_certificate  =  local.users[count.index].cert
+#}
```

Comment:
Can you remove all of this commented out code. The complete example needs to be updated to align with the complete example for postgres https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/examples/complete/main.tf